content/automated-rust-releases.md

```diff
@@ -0,0 +1,582 @@
+
+> [**git-cliff**](https://github.com/orhun/git-cliff) is a command-line tool (written in [Rust](https://www.rust-lang.org/)) that provides a highly customizable way to generate changelogs from git history. It supports using [custom regular expressions](/docs/configuration#commit_parsers) to alter changelogs which are mostly based on [conventional commits](/docs/configuration#conventional_commits). With a single [configuration file](/docs/configuration), a big variety of formats can be applied for a changelog, thanks to the Jinja2/Django-inspired [template engine](/docs/category/templating). More information and examples can be found in the [GitHub repository](https://github.com/orhun/git-cliff).
+
+For our changelog format, we can simply go with the default configuration which outputs something like the following:
+
+```sh
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [unreleased]
+
+### Documentation
+
+- Add link to emacs package support git-cliff (#307)
+```
+
+For this configuration, first we should install `git-cliff`:
+
+```sh
+cargo install git-cliff
+```
+
+Then we can initialize it as follows:
+
+```sh
+git cliff --init
+```
+
+This will create `cliff.toml` in the current directory and you can commit it in the root directory of your repo for `release-plz` to pick it up.
+
+If you want to generate a fancier changelog, you can look at the [examples](https://git-cliff.org/docs/templating/examples) or use the [configuration that `git-cliff` uses](https://github.com/orhun/git-cliff/blob/main/cliff.toml) which corresponds to the following:
+
+```sh
+$ git cliff
+
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [unreleased]
+
+### 📚 Documentation
+
+- _(readme)_ Add link to emacs package support git-cliff ([#307](https://github.com/orhun/git-cliff/issues/307)) - ([fa471c7](https://github.com/orhun/git-cliff/commit/fa471c7178dce184ca6fe5bbb24b9c2db96d68ce))
+```
+
+Great, now we are ready to set up the actual releaser tool!
+
+---
+
+### **release-plz**
+
+> [release-plz](https://release-plz.ieni.dev) creates a fully automated release pipeline, allowing you to easily release changes more frequently, without the fear of doing typos or other subtle manual mistakes you can make when releasing from your terminal.
+
+Unlike `git-cliff`, we don't actually need to install `release-plz` as a command-line tool. It is because `release-plz` runs from the CI (GitHub Actions) and we are not doing anything locally for the releases. However, we still need to tweak some things regarding how we want to release our project.
+
+For configuring `release-plz`, create `release-plz.toml` with the following contents at the root of your repository:
+
+```toml
+[workspace]
+# path of the git-cliff configuration
+changelog_config = "cliff.toml"
+
+# enable changelog updates
+changelog_update = true
```

Comment:
`changelog_update`, `git_tag_enable`, `allow_dirty`, `publish_allow_dirty` are the default values. Why do you override them? 🤔

Reply:
Just to show the reader that they are available options :)